birdhouse/components/stac/backup

```diff
@@ -20,9 +20,10 @@ backup_representative() {
                --network "${COMPOSE_PROJECT_NAME}_default" \
                -v ${BIRDHOUSE_BACKUP_VOLUME}:/backup \
                --user root \
+               --env BIRDHOUSE_LOG_LEVEL=${BIRDHOUSE_LOG_LEVEL} \
                --entrypoint sh \
                ${STAC_POPULATOR_BACKUP_IMAGE} \
-               -c 'mkdir /backup/stac-repr/; stac-populator export http://stac:8000/stac /backup/stac-repr/ --ignore-duplicate-ids'
+               -c 'mkdir /backup/stac-repr/; stac-populator --log-level-stderr ${BIRDHOUSE_LOG_LEVEL} export http://stac:8000/stac /backup/stac-repr/ --ignore-duplicate-ids'
```

Comment:
My only worry here is that the levels may not necessarily match in the future. For example, we discussed at once point adding a TRACE level which would be an invalid log level for the other command.

What if instead we made the log level in stac-populator settable using an environment variable and then passed that variable to this docker container.

Reply:
We don't actually need setting from the env variable. Here, we're directly using the variable of the current environment (where the CLI is ran from) and propagating it to the tool's CLI option. 

The assumption/logic here is that, if the user adds log-level options to increase debugging of the `birdhouse backup ...` commands, they are (as I was) probably trying to debug what is happening, and not having the `stac-populator`'s debug details as well doesn't help further in debugging. There could be a separate `STAC_POPULATOR_LOG_LEVEL` variable to control them separately, but that is not really useful when using the `birdhouse --log-level` flag directly.